sds_data_manager/constructs/batch_compute_resources.py

```diff
@@ -48,6 +51,10 @@ def __init__(
             S3 bucket.
         repo : ecr.Repository
             Container repo
+        batch_security_group: ec2.SecurityGroup
+            Batch security group
+        db_secret_name : str
```

Comment:
```suggestion
        rds_secret_name : str
```
Since you're calling it rds_secret below it'd be good to be consistent in the naming everywhere.